script.js

```diff
@@ -1,12 +1,375 @@
-//You can edit ALL of the code here
+window.onload = setup
+
+const shows = getAllShows()
+let allEpisodes
+let selectedShowId
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+    loadAllShows()
+    makePageForShows()
+    searchFunc()
+}
+
+const rootElem = document.getElementById('root')
+    //a container for all episodes
+let divContainer = createHtmlElement('div')
+divContainer.className = 'container'
+    //Create and append select show list
+let showList = createHtmlElement('select')
+showList.className = 'showSelect'
```

Comment:
you can put some element straight to html since is not dynamic

Reply:
a good point  will keep it in mind thanks